src/file-picker/src/FilePicker.js

```diff
@@ -98,6 +104,13 @@ export default class FilePicker extends PureComponent {
       buttonText = 'Replace files'
     }
 
+    let placeholderText
+    if (placeholder) {
+      placeholderText = placeholder
+    } else {
+      placeholderText = 'Select a file to upload…'
+    }
```

Comment:
This could be replaced by setting a defaultProp instead.

```js
static defaultProps = {
  placeholder: 'Select a file to upload…'
  // ... other stuff
}
```

This would allow you to simply pass the prop as-is to the `<TextInput placeholder={placeholder}`

Reply:
Oh sure! It's fixed now!